prow/cmd/deck/main.go

```diff
@@ -537,7 +541,11 @@ func renderSpyglass(sg *spyglass.Spyglass, ca *config.Agent, src string, o optio
 		}
 	}
 
-	lenses := sg.Views(viewerCache)
+	ls := sg.Lenses(viewerCache)
```

Comment:
Are we going to leave the viewer terminology here?

Reply:
I was not entirely thorough in killing references to `viewer`; I mostly only changed the ones I happened to be touching.

I don't really like the `viewerCache` naming (it is confusing to me), and renamed similar variables elsewhere.

If I were to change the rest of the `viewer` references, I'd probably want to do so in a separate PR, to avoid inflating this one more than it already is.

Reply:
🙏